src/Symfony/Component/Scheduler/Event/PostRunEvent.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Scheduler\Event;
+
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Scheduler\Generator\MessageGenerator;
+
+class PostRunEvent
+{
+    public function __construct(private readonly MessageGenerator $messageGenerator, private readonly Envelope $envelope)
```

Comment:
Assuming we dispatch these events in the scheduler itself, we cannot use `Envelope` here as it could be any object.

Reply:
Indeed the standalone usage